qemu/tests/cfg/multi_disk_wild_hotplug.cfg

```diff
@@ -0,0 +1,26 @@
+- multi_disk_wild_hotplug:
+    type = multi_disk_wild_hotplug
+    only virtio_scsi,virtio_blk
+    only Linux
+    start_vm = no
+    not_preprocess = yes
+    force_create_image = yes
+    force_create_image_image1 = no
+    remove_image = yes
+    remove_image_image1 = no
+    stg_image_name = "images/stg%s"
+    stg_image_num = 41
+    stg_image_size = 256M
+    repeat_num = 300
+    unplug_time = 3
+    plug_time = 3
+    virtio_scsi:
+      guest_operation = _run_sg_luns
+    virtio_blk:
+      repeat_num = 100
+      q35:
```

Comment:
`22:46:39 WARNI| Ignore hotplug error: QMP command 'device_add' failed    (arguments: OrderedDict([('driver', 'virtio-blk-pci'), ('id', 'stg1'), ('drive', 'drive_stg1'), ('bootindex', '2'), ('write-cache', 'on'), ('bus', 'pcie-root-port-5'), ('addr', '0x0'), ('iothread', 'iothread0')]),    error message: {'class': 'GenericError', 'desc': "Duplicate ID 'stg1' for device"})`

need add ARM support
`q35, arm64-pci, arm64-mmio:`